src/helpers/donors.js

```diff
@@ -0,0 +1,10 @@
+
+export default {
+  'steemit': 'gold',
```

Comment:
I used the single quote because it would not work for username with - or . character